includes/class-admin.php

```diff
@@ -144,15 +144,6 @@ public static function register_settings() {
 				'default'      => array( 'post', 'pages' ),
 			)
 		);
-		// \register_setting(
```

Comment:
I was going to go with your suggestion, where a filter would be required to display the option